scripts/test.sh

```diff
@@ -72,7 +73,7 @@ function _unittest_with_coverage {
     for import_path in ${PACKAGES}; do
         package=$(basename ${import_path})
 
-        go test -v -race -covermode=$covermode                                   \
+        go test -mod=vendor -v -race -covermode=$covermode                                   \
```

Comment:
Looks like the `\` spacing got messed up. Should be lined up with the other slashes.